css/index.css

```diff
@@ -1,3 +1,450 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+ 
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+h2 {
+  font-family: "Anton", sans-serif;
+}
+
+.static-navbar {
+  /* position: fixed; */
+  top: 0;
+  width: 100%;
+  /* background-color: rgb(0, 93, 189); */
+  z-index: 5;
+}
+
+nav {
+  display: block;
+  width: 880px;
+  height: 100px;
+  margin: 0 auto;
+  position: relative;
+}
+
+nav a {
+  text-decoration: none;
+  color: rgb(150, 144, 144);
+  margin: 35px 31px 0 31px;
+  font-size: 14px;
+  display: inline-block;
+}
+
+nav a:hover {
+  color: rgb(176, 177, 178);
+}
+
+nav a:first-child {
+  margin-left: 0;
+}
+
+nav div {
+  display: inline-block;
+  position: absolute;
+  right: 0;
+}
+
+img {
+  margin-top: 25px;
+}
+
+.website-wrapper {
+  width: 880px;
+  margin: 130px auto 0 auto;
+  z-index: 1;
+}
+
+header {
+  border-bottom: 1px solid black;
+  position: relative;
+  height: 490px;
+}
+
+header div {
+  display: inline-block;
+  text-align: center;
+  margin: 70px 0 0 85px;
+}
+
+header div h1 {
+  font-size: 80px;
+  margin-bottom: 30px;
+}
+
+header div h1 span {
+  display: block;
+}
+
+header div a {
+  text-decoration: none;
+  color: black;
+  border: 1px solid black;
+  padding: 4px 48px;
+}
+
+header img {
+  display: inline-block;
+  position: absolute;
+  right: 0;
+}
+
+.section-one {
+  margin-top: 39px;
+}
+
+.section-one div {
```

Comment:
now that we are exploring more responsive elements we can use more % based solutions and add those rem and em